ctapipe/reco/shower_reconstructor.py

```diff
@@ -0,0 +1,165 @@
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import AltAz, SkyCoord
+from iminuit import Minuit
+from scipy.stats import poisson
+
+from ctapipe.containers import ReconstructedGeometryContainer
+from ctapipe.coordinates import CameraFrame, TelescopeFrame
+from ctapipe.core.traits import Unicode
+from ctapipe.image import GaussianShowermodel, ShowermodelPredictor
+from ctapipe.reco import Reconstructor
+
+
+class ShowerReconstructor(Reconstructor):
```

Comment:
Following the new scheme, I think `Model3DGeometryReconstuctor` would be correct (and it would then be a subclass of `GeometryReconstructor`.   The word "shower" is not needed, as it is always the case that we are reconstructing showers, so it's redundant (we don't say ShowerHillasReconstructor, for example)